doc/changes/latest.inc

```diff
@@ -59,6 +59,9 @@ Bugs
 - Fix bug with :func:`mne.io.read_raw_fil` where datasets without sensor positions would not import (:gh:`11733` by `George O'Neill`_)
 - Fix bug with :func:`mne.chpi.compute_chpi_snr` where cHPI being off for part of the recording or bad channels being defined led to an error or incorrect behavior (:gh:`11754`, :gh:`11755` by `Eric Larson`_)
 - Allow int-like for the argument ``id`` of `~mne.make_fixed_length_events` (:gh:`11748` by `Mathieu Scheltienne`_)
+- Fix bugs with :mod:`mne.decoding` classes (:gh:`11846` by `Eric Larson`_):
+  - :class:`mne.decoding.ReceptiveField` now sets ``rf.fit_intercept_`` during ``fit()`` rather than overwriting ``rf.fit_intercept``
+  - :class:`mne.decoding.SlidingEstimator` now returns the same ``dtype`` as ``X`` (usually ``float64``) in ``sl.predict(X)`` rather than always returning ``int``
```

Comment:
I guess you meant sl.transform as predict should still return int if you do classification.

Reply:
Yes and it does, I'll fix it!